library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/utils/MutationDataForgeryFactory.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.utils
+
+import com.datadog.android.sessionreplay.model.MobileSegment
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.ForgeryFactory
+
+internal class MutationDataForgeryFactory :
+    ForgeryFactory<MobileSegment.MobileIncrementalData.MobileMutationData> {
+    override fun getForgery(forge: Forge): MobileSegment.MobileIncrementalData.MobileMutationData {
+        return MobileSegment.MobileIncrementalData.MobileMutationData(
+            adds = forge.aNullable {
```

Comment:
why it is nullable when it is `required` in the [schema](https://github.com/DataDog/rum-events-format/blob/2fb02f9a1eb215290f398b71ae30eafb10c11964/schemas/session-replay/mobile/mutation-payload-schema.json#L7) and 2 other fields `removes` and `updates` below are not nullable it seems?

Reply:
Good point ;)